### DISCUSSION TITLE
Error message in the front section of file uploading jar package

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#881] [Q&A]
![image](https://user-images.githubusercontent.com/50607317/175484389-93795376-fb7b-439a-a617-2fd41f98ed76.png)
why?Upload .py file is OK

@junoyoon

**Comment by junoyoon** [comment]
Please let us know the size of jar file?

**Reply by zhougy1989** [reply]
The smallest jar package is only 27KB

**Reply by zhougy1989** [reply]
The previous version 3.4 can also be uploaded. I now use 3.5.5

**Reply by junoyoon** [reply]
I have no idea about what your env is.
Please give us more details like

1. Whether you used tomcat or not..
2. The url you hide in above picture.
3. Whether you access the server in local PC or in remote PC.
4. Whether you configured http proxy in your browser or not
5. If you have configured proxy in browser..  It might be that your company's firewall blocks jar file. So tell us your company firewalls policy.
